config/webhook/manifests.yaml

```diff
@@ -93,6 +93,26 @@ webhooks:
     resources:
     - portals
   sideEffects: None
+- admissionReviewVersions:
```

Comment:
Please update the base to the latest, the changes here are already included in the base

Reply:
Yes, I also saw those changes. It seems a duplicate was made when I pulled the project. The change is now fixed!